diff -durN xroar-0.28/Makefile xroar-0.28-lm/Makefile
--- xroar-0.28/Makefile	2012-05-12 09:01:33.000000000 +0200
+++ xroar-0.28-lm/Makefile	2012-05-13 10:44:42.265766639 +0200
@@ -335,7 +335,7 @@
 	-I$(CURDIR) -I$(SRCROOT) $(WARN) \
         -DVERSION=\"$(VERSION)\" \
         -DROMPATH=$(ROMPATH) -DCONFPATH=$(CONFPATH)
-xroar_unix_LDFLAGS = $(LDFLAGS) $(LDLIBS) $(xroar_opt_LDFLAGS)
+xroar_unix_LDFLAGS = $(LDFLAGS) $(LDLIBS) $(xroar_opt_LDFLAGS) -lm
 
 portalib_CFLAGS = $(CFLAGS) $(CPPFLAGS) \
 	-I$(CURDIR) -I$(SRCROOT) $(WARN)
