diff -up xroar-0.27/Makefile~ xroar-0.27/Makefile
--- xroar-0.27/Makefile~	2011-09-27 09:33:38.000000000 +0200
+++ xroar-0.27/Makefile	2011-10-02 11:13:26.507106375 +0200
@@ -308,7 +308,7 @@ xroar_unix_OBJCFLAGS = $(OBJCFLAGS) $(CP
 	-I$(CURDIR) -I$(SRCROOT) $(WARN) \
         -DVERSION=\"$(VERSION)\" \
         -DROMPATH=$(ROMPATH) -DCONFPATH=$(CONFPATH)
-xroar_unix_LDFLAGS = $(LDFLAGS) $(LDLIBS) $(xroar_opt_LDFLAGS)
+xroar_unix_LDFLAGS = $(LDFLAGS) $(LDLIBS) $(xroar_opt_LDFLAGS) -lm
 
 xroar_unix_ALL_OBJS = $(xroar_common_OBJS) $(xroar_common_INT_OBJS) \
 	$(xroar_unix_OBJS) $(xroar_unix_INT_OBJS) \
